modules/pool/subgraph-mapper.ts

```diff
@@ -0,0 +1,289 @@
+import { Chain, PrismaPoolType } from '@prisma/client';
+import { BalancerPoolFragment } from '../subgraphs/balancer-subgraph/generated/balancer-subgraph-types';
+import { AddressZero } from '@ethersproject/constants';
+import * as dataMappers from './pool-data';
```

Comment:
absolutely, we have the option to individually import all the dataMappers. before I change that, let me ask what you think. my rationale for using a namespace is to keep it clear where these functions are coming from. especially around handling types. to make it work with individual imports i'd probably need to add a suffix, like `fxDataMapper` and it would end up quite similar to the current `dataMappers.fx`.

Reply:
updated to have all the imported functions in the header.